src/main/java/uk/co/ramp/ContactRunner.java

```diff
@@ -25,13 +26,10 @@
 
     private static final Logger LOGGER = LogManager.getLogger(ContactRunner.class);
     public static final String COMPARTMENTS_CSV = "Compartments.csv";
-
+    private InputFiles inputFiles;
```

Comment:
Might be better to make it explicit that `InputFiles` retrieves the file location rather than the actual settings/data

```suggestion
        try (Reader reader = new FileReader(inputFileLocation.contactData())) {
```